source/NVDAObjects/UIA/__init__.py

```diff
@@ -824,6 +825,16 @@ def _get_shouldAllowUIAFocusEvent(self):
 		except COMError:
 			return True
 
+	_lastLiveRegionChangeInfo=(None,None) #: Keeps track of the last live region change (text, time)
+	def _get__shouldAllowUIALiveRegionChangeEvent(self):

```

Comment:
It'd be good to have a brief comment here explaining what this does, similar to your commit summary. Not that the code is complex, but super clarity is nice. :)